app/presenters/shopping_cart_presenter.rb

```diff
@@ -1,14 +1,18 @@
 # frozen_string_literal: true
 
 class ShoppingCartPresenter
-  def initialize(products)
-    @products = products
+  def initialize(cart_items)
+    @cart_items = cart_items
   end
 
   def products_in_cart
-    return 'Your shopping cart is empty.' if @products.empty?
+    return 'Your shopping cart is empty.' if @cart_items.empty?
 
-    products_number = @products.uniq.count
+    products_number = @cart_items.uniq.count
```

Comment:
wydaje mi się, że zamiast products_numeber lepszą nazwą będzie products_count -> ale to komsetyka :)